python/cuml/tests/dask/test_dask_tsvd.py

```diff
@@ -28,12 +28,14 @@
 )
 @pytest.mark.parametrize("input_type", ["dataframe", "array"])
 def test_pca_fit(data_info, input_type, client):
+    # Assume at least 4GB memory
+    max_gpu_memory = pytest.max_gpu_memory if pytest.max_gpu_memory else 4
```

Comment:
If you really mean "at least 4GB", then it should be something like:

```suggestion
    # Assume at least 4GB memory
    max_gpu_memory = max(pytest.max_gpu_memory or 0, 4)
```

But I think we really mean this:
```suggestion
    # Utilize max memory (fallback to 4GB)
    max_gpu_memory = pytest.max_gpu_memory or 4
```